mathjax3-ts/output/chtml/Wrappers/mo.ts

```diff
@@ -38,7 +38,7 @@ export class CHTMLmo extends CHTMLWrapper {
      * The font size that a stretched operator uses.
      * If -1, then stretch arbitrarily, and bbox gives the actual height, depth, width
      */
-    protected size: number = null;
+    public size: number = null;
```

Comment:
Should probably be private with a getter/setter.

Reply:
Why the overhead, if it's just going to do the same thing as reading (or setting) the value anyway?  Should there come a need to do something algorithmic, we can add 

```
protected _size: number;

get size() {return this._size}
set size(size) {this._size = size; return size}
```

or whatever is needed at that time, and nothing external that uses `size` will have to change.  But why add that overhead now for no reason?